src/components/Courses/Courses.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from "react";
+import axios from 'axios';
+
+import classes from "./Courses.module.css";
+
+const Courses = (props) => {
+
+  // Initial state for tracking catalogs
+  const [coursesState, setCoursesState] = useState({
+    courses: null,
+    isLoading: false,
+    error: null
+  });
+
+  // TODO: Replace this DAU call with the selected in catalog name
+  const catalog_courses_api_url = process.env
+    .REACT_APP_XIS_COMPOSITELEDGER_API + "?provider=DAU";
+  
+  let coursesLength = 0;
+
+  let result = (
+    <div>
+      Error loading courses. Please contact an administrator.
```

Comment:
Sometimes we say "courses", sometimes we say "experiences" -- we should be consistent in our terminology.

Reply:
Let's go with courses.